test/e2e/snaps/test-snap-jsx.spec.js

```diff
@@ -35,8 +34,8 @@ describe('Test Snap JSX', function () {
         await driver.waitForSelector('#connectjsx');
         await driver.clickElement('#connectjsx');
 
-        // switch to metamask extension and click connect
-        await switchToNotificationWindow(driver, 2);
```

Comment:
`switchToNotificationWindow` is deprecated, so they've been removed as a bit of a drive-by refactor.